spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
please add the following to the `.sync.yml`:
```
---
spec/spec_helper_acceptance.rb:
  unmanaged: false
```

Reply:
done.